packages/core/src/textures/BaseTexture.ts

```diff
@@ -72,32 +72,32 @@ export class BaseTexture<R extends Resource = Resource, RO = IAutoDetectOptions>
      * @member {PIXI.ALPHA_MODES}
      * @default PIXI.ALPHA_MODES.UNPACK
      */
-    public alphaMode?: ALPHA_MODES;
+    public alphaMode: ALPHA_MODES;
```

Comment:
One small request, could you create another PR for the optional members? Seems like a separate issue.

Reply:
It triggered a bunch of strict type warning, so I included it here. But I can split it off.

Reply:
Yeah, it's helpful when we look back or have to revert to have PRs be single ideas.